content/en/blog/kairos-meets-k0s.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Kairos Meets k0s: A Meta-Distribution for Kubernetes is Born"
+date: 2025-04-01T10:53:13+01:00
+author: Mauro Morales ([LinkedIn](https://www.linkedin.com/in/mauromorales/)) ([GitHub](https://github.com/mauromorales))
+---
+
+Up until now, Kairos has shipped with first-class support for k3s out of the box. If you wanted something else? You had to rely on community-powered providers. And let’s be honest—while amazing in their own right, they’ve been... fragmented.
```

Comment:
is there a dash in there between honest-while or its some markdown?

Reply:
@Itxaka yeah it's an em dash: generaly used to note a break, provide drama, or give an example.